src/components/DetailView/vegaSpec.js

```diff
@@ -147,6 +145,17 @@ export function createSpec(sensor, primaryValue, selections, initialSelection) {
             },
             encoding: {
               color: colorEncoding(selections),
+              x: {
```

Comment:
what is the difference to `xDateRangeEncoding` ? at a first glance, I cannot spot it

Reply:
You're right.  It evolved over several steps, so I'll make it the same here.  

But for the next layer, tickCount is 'day', and the whole axis property is repeated as a consequence.  Maye I can just use axis: {...(xDateRangeEncoding.axis), tickCount: 'day'}.  I'll try it.

Reply:
Hang on... this change of Detailview/vegaSpec.js is supposed to be part of a different changeset:   https://github.com/cmu-delphi/www-covidcast/pull/574
Don't know how they got mixed up but maybe I forgot to switch branches.  Sigh.  Now I have to figure out how to uncommit this part, and/or unpush it, or whatever.

Reply:
> This merge of branch 'dev' into dlaliberte/#563 appears to be the reason why my changes to DetailView/vegaSpec.js are part of this change. How do I undo that?

there is no easy option, the simple option is when it happens that you say in the summary of the PR that it is based on another one. and then we have to make sure that the other one is merged first

Reply:
Apparently I did this before as well, since you noted in a comment that my change included another one.  So I am doing something wrong, but I don't know what I did in order to avoid doing it again.    For this change, I am going to start over with another branch, and avoid "synching" until I am forced to.